hera_cal/vis_clean.py

```diff
@@ -825,6 +825,7 @@ def zeropad_array(data, binvals=None, zeropad=0, axis=-1, undo=False):
             if undo:
                 s = [slice(None) for j in range(data.ndim)]
                 s[ax] = slice(zeropad[i], -zeropad[i])
+                s = tuple(s)
```

Comment:
If we don't do this, a warning is generated:
```
Using a non-tuple sequence for multidimensional indexing is
deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the
future this will be interpreted as an array index, `arr[np.array(seq)]`,
which will result either in an error or a different result.
```
I can't do a tuple comprehension, because tuples are immutable, and so the elements can't be changed after it's created (as in the line immediately prior).